simpledotorg/simple-server
# Use materialized view for patient breakdown card

**Story card:** [sc-7863](https://app.shortcut.com/simpledotorg/story/7863/the-total-registrations-on-the-overview-page-and-the-detail-page-don-t-match)

## Because

The population coverage card is populated by directly querying the patient table. This causes inconsistency in the dashboard.

## This addresses

The population coverage card is now populated from the materialized view.

## Some other changes

Extract the `Lost to follow-up` and `Population coverage` partials  into view components

## Test instructions

- Checkout to this branch
- Go to reports > `<region>` > Details subsection in Hypertension/Diabetes
- Verify that hypertension/diabetes coverage card is consistent with the numbers on other cards.
